Tests/SwiftParserTest/translated/ImplicitGetterIncompleteTests.swift

```diff
@@ -43,8 +43,16 @@ final class ImplicitGetterIncompleteTests: XCTestCase {
       diagnostics: [
         DiagnosticSpec(message: "expected '}' to end variable", fixIts: ["insert '}'"]),
         DiagnosticSpec(message: "expected '}' to end function", fixIts: ["insert '}'"]),
-      ]
+      ],
+      fixedSource: #"""
+        // Would trigger assertion when AST verifier checks source ranges ("child source range not contained within its parent")
+        func test2() {
+          var a : Int {
+            switch i {
+        }
```

Comment:
It would be nice if these braces were properly indented.

Reply:
https://github.com/apple/swift-syntax/issues/1601